subsys/nrf_security/src/drivers/cracen/cracenpsa/src/common.c

```diff
@@ -835,13 +837,18 @@ size_t cracen_get_opaque_size(const psa_key_attributes_t *attributes)
 		if (PSA_KEY_TYPE_IS_ECC(psa_get_key_type(attributes))) {
 			if (psa_get_key_type(attributes) ==
 			    PSA_KEY_TYPE_ECC_PUBLIC_KEY(PSA_ECC_FAMILY_SECP_R1)) {
-				return PSA_EXPORT_PUBLIC_KEY_OUTPUT_SIZE(
+				*key_data = PSA_EXPORT_PUBLIC_KEY_OUTPUT_SIZE(
```

Comment:
Is this meant to be key_size? Otherwise where is key_data from?

Reply:
Thanks! It seems that I commited the file in the middle of a rename. Should be fixed now :)